mathics/core/expression.py

```diff
@@ -1155,6 +1167,21 @@ def block(tex, only_subsup=False):
         else:
             raise BoxError(self, 'tex')
 
+    def output_cost(self):
+        name = self.get_head_name()
+
+        if name in ('System`ImageBox', 'System`GraphicsBox', 'System`Graphics3DBox'):
+            return 0  # always display
+
+        cost = sum(leaf.output_cost() for leaf in self.leaves)
+
+        if name == 'System`List':
+            return 2 + cost + len(self.leaves)  # {a, b, c}
```

Comment:
would it be better to compute the sum cost of leaves? Also, what about the size of ", " between leaves?